src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Version>4.0.2</Version>
```

Comment:
Did you verify that Version actually works for the package? Because in the latest repos we changed it to be `<PackageVersion>` could you please confirm that?

Reply:
Yes, the packages it builds locally are versioned as I would expect.

Reply:
Thanks for validating.